### ISSUE TITLE
Node/nodejs Twitch account

### ISSUE THREAD

**Issue by bnb** [nodejs/evangelism#91]
So, now that we're under nodejs/evangelism, are we able to officially act on behalf of the Node foundation? I would like to send a message to the Twitch team, asking them if they would release either Twitch.tv/node or Twitch.tv/nodejs to us. Fairly confident that they will if we ask nicely. Also, if the Foundation owns the Node(.js?) trademark, that might be useful.

If the answer is yes, which would you prefer?

**Comment by mikeal** [comment]
What would we use the Twitch account for?

**Comment by diagramatics** [comment]
I guess... game development with Node.js? Any other development outside of games would be [prohibited](http://www.twitch.tv/p/rules-of-conduct).

**Comment by bnb** [comment]
Meetings, development, and community program-a-longs. Despite the Rules of Conduct @diagramatics linked to, there are many people, including prominent developers, who stream [Programming](http://www.twitch.tv/directory/game/Programming) and live talk shows. Program-a-longs would be an excellent way to engage with the community - essentially, live tutorials where questions can be asked. We could even have third parties run progam-a-longs on the official channel that show how to use their APIs.

I would be happy to start an open dialogue with Twitch to see if they're willing to allow it. I am quite certain that they will be very open to it.

**Comment by bnb** [comment]
After reading https://github.com/nodejs/membership/issues/10, I came up with another reason to have a Twitch channel: fundraising and charity events. There are many gaming livestreams and events that take donations for charity, Operation Supply Drop is the one I can think of off the top of my head. 

The way these types of things generally go is that the org sets up a goal, and places rewards for both donations and milestones. It's pretty much exactly like a Kickstarter with rewards for donations and stretch goals that are built into the original goal.

It would be amazing to do charity events where we donated the proceeds to non-profits who are actively teaching children, the less fortunate, etc. to code.

The Node Foundation could also run community fundraisers for costs such as contributor compensation, servers, service purchases, and so on - I've seen this type of thing on Twitch as well.

We don't need to get bogged down by thinking that Twitch is only for games. I am very confident they would welcome Node in an official, non-gaming role. All we have to do is reach out. I don't know if my role in the evangelism WG is high enough for me to contact them at an official manner, so I just want to get approval from others first.
